companydotcom/vastly
# ATX-2466 | adds storybook to `ui` and `forms`, links them locally

- creates stories for all components in `ui` and `forms`
- organizes file structure
- slight styling of docs
- init `Getting Started` page

To Test: 
1. Navigate into `ui` in one terminal and `forms` in another terminal
2. run `pnpm run storybook` in both terminals
3. Once the browsers have opened, go the the `localhost:6006` page to see both storybooks in once place

TODO: 
Once both storybooks have a static page deploy, we can replace the url in `ui` to the proper forms storybook url
